src/goOutline.ts

```diff
@@ -51,31 +51,20 @@ export interface GoOutlineOptions {
 	 */
 	document?: vscode.TextDocument;
 
-	/**
-	 * Skips range information in the output.
-	 * Calculating ranges is slightly expensive for large files, therefore skip it when not required.
-	 */
-	skipRanges?: boolean;
```

Comment:
Why have we deleted the `skipRanges` option?

Reply:
@ramya-rao-a  the `DocumentSymbol` it wont accept null paramters for ranges ( i.e in needs ranges).

Reply:
was this something we needed ? from what I could tell we were not actually skipping ranges in the provider

Reply:
Currently that option is being used in only in the `Go: Add Imports` command which doesnt need the range.

The only benefit is we avoid the work involved in calculating the ranges.

Reply:
doesn't seem to be a way to avoid the ranges as it throws if those are null.